STEER/STEERBase/AliMagFastDip5k/AliMagFastDip5k.cxx

```diff
@@ -0,0 +1,11 @@
+#include "AliMagFast.h"
+#include "AliMagFastDip5k.h"
+typedef unsigned short ushort;
+const float Infinity = INFINITY;
+AliMagFast::SegmentEnd_t dip5k_z_slices[] = {
+};
+AliMagFast::SegmentSearch_t dip5k_z_segments[] = {
+};
+AliMagFast::SegmentSearch_t dip5k_z = {-1, 0.0, -1760.0, dip5k_z_slices, dip5k_z_segments};
```

Comment:
@syamasak building on ubuntu I get an error here: 
````
/home/shahoian/alice/sw/SOURCES/AliRoot/0/0/STEER/STEERBase/AliMagFastDip5k/AliMagFastDip5k.cxx:9:90: error: narrowing conversion of ‘-1’ from ‘int’ to ‘UShort_t {aka short unsigned int}’ inside { } [-Wnarrowing]
 AliMagFast::SegmentSearch_t dip5k_z = {-1, 0.0, -1760.0, dip5k_z_slices, dip5k_z_segments};
                                                                                          ^
/home/shahoian/alice/sw/SOURCES/AliRoot/0/0/STEER/STEERBase/AliMagFastDip2k/AliMagFastDip2k.cxx:9:90: error: narrowing conversion of ‘-1’ from ‘int’ to ‘UShort_t {aka short unsigned int}’ inside { } [-Wnarrowing]
 AliMagFast::SegmentSearch_t dip2k_z = {-1, 0.0, -1760.0, dip2k_z_slices, dip2k_z_segments};
                                                                                          ^
````
Indeed, the SegmentSearch_t is defined as 
````
  struct SegmentSearch {
    UShort_t nDivision;
    float factor;
    float offset;
    SegmentEnd_t *slices;
    SegmentSearch *segments;
  };
````
I guess the initialization should be done as  ````dip5k_z = {0, ...````